api_app/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.4.54"
+__version__ = "0.4.57"
```

Comment:
Commenting here as there's no better place for this:
1. Can you update the changelog
2. [nit] I think it's best to use a vector image (and not a raster one). It should be simple enough to find an SVG version of it...

Reply:
Yep good shout thanks, have addressed both